doc/gov/chapters/chapter04.md

```diff
@@ -45,51 +45,51 @@ Any Workstream Leads or Co-Leads positions can be held by any Participant.
 ### 4.2.3 Nomination
 
 - Any individual can nominate themselves or another participant for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
-- An individual can self-nominate themselves or another participant to multiple positions.
+- An individual can nominate themselves or another participant to multiple positions.
 
 <a name="4.2.4"></a>
 ### 4.2.4 Selection
 
-The process of nomination and selection is owned and operated by CNTT with the help from LFN staff.
+The process of nomination and selection is owned and managed by CNTT with help from LFN staff.
 
 There will be a nomination window (typically around February each year) during which individuals can nominate others or nominate themselves for a position. The exact start and end date of the nomination window will be determined by Technical or Governance Committee Leads(depending on the nature of the Workstream).
 
-During the nomination period, Participants can send their nominations to a designated individual who will help in administering the process (from CNTT Governance team or LFN Staff) . The details and email addresses of that individual will be clarified when the process is started.
+During the nomination period, Participants can send their nominations to a designated individual who will help in administering the process (from CNTT Governance team or LFN Staff). The details and email addresses of that individual will be confirmed at the begining of the process.
 
-Once the nomination period expires for a particular Workstream, the selection process will start as follows:
+Once the nomination period ends, the selection process for the new leads and co-leads for a given workstream is as follows:
 
-- Selection process for Leads positions and Co-Leads will occur separately.
-- No voting will be required if number of nominations received for a particular position is less than or equal to the number of available vacancies. 
-- Online Voting process will take place to select candidate(s) from the nominee list for any given position.
-  - The exact tool used for voting will be determined by the LFN Staff who will help in administering the process and will depends on the approved tool at that point of time.
+- The selection process for Leads positions and Co-Leads will occur separately.
+- No voting is required if the number of nominations received for a particular position is less than or equal to the number of available vacancies. 
+- An online voting process will take place to select candidate(s) from the nominee list for each position.
+  - The exact tool used for voting will be determined by the LFN Staff and depends on the approved tool at time of voting.
 - Only Voting representatives can vote (from the up to date voting representative list).
-- Candidate(s) who receives the highest number of votes win.
+- The candidate(s) who receives the highest number of votes win.
```

Comment:
What happens if there is only one position and there is a tie in the number of votes each candidate received.